### PR TITLE
Chatbot Response class Handling

### PR DESCRIPTION
**Pull request by prasanthhs**
Whats new: Improved code readability for input and output handling, Easy to customise and formulate UI Response for new developers.
Functionality :  Same as before. No new functionality.

### CODE REVIEW

**Comment by RicardoUsbeck** [comment]
Can you tell me more about "Update of ReadMe for WARNING OF THIS UPDATE" and how this was solved?

**Comment by prasanthhs** [comment]
@RicardoUsbeck That was the first version Warning. I had updated so that others would not use this code base in case they see a check in. Now all the related code is checked in. Please check "UI updates to support new Response class" version. I had repudiated ReadMe removing the warning.

**Comment by RicardoUsbeck** [comment]
Okay, I want to have that checked by the reviewers, and then we can merge the branch

File: chatbot/src/main/java/chatbot/core/handlers/eliza/ElizaHandler.java [hunk 1]
```diff
@@ -13,18 +16,27 @@
...
 			                      .getText();
 			ElizaMain eliza = new ElizaMain();
 			eliza.readScript(true, "src/main/resources/eliza/script");
 			String response = eliza.processInput(query);
 			log.info("search:Got Response , response= " + response);
-			String responseJSON = "[{ \"comment\":\"" + response + "\"}]";
-			return responseJSON;
+			responselist.setMessageType(MessageType.PLAIN_TEXT);
+			responselist.addMessage(createNewResponse(response)); // Add Response to Final String
+			//String responseJSON = "[{ \"comment\":\"" + response + "\"}]";
```
Comment (sepidetari) [inline chatbot/src/main/java/chatbot/core/handlers/eliza/ElizaHandler.java]: responseJSON seems not to be needed anymore so this line can be deleted

File: chatbot/src/main/java/chatbot/core/handlers/eliza/ElizaHandler.java [hunk 2]
```diff
@@ -13,18 +16,27 @@
...
+			return responselist;
 		}
 		/*catch (JsonProcessingException e) {
```
Comment (sepidetari) [inline chatbot/src/main/java/chatbot/core/handlers/eliza/ElizaHandler.java]: The search method throws JsonProcessingException and IOException but here it is commented out..Is it supposed to be added later?

Comment (prasanthhs) [reply chatbot/src/main/java/chatbot/core/handlers/eliza/ElizaHandler.java]: @sepidetari  They dont get thrown by this class so it is not required to handle those 2 Exceptions.

**Comment by prasanthhs** [comment]
@sepidetari @Suganya31 Thank you for your review. Yes i agree about the lack of functional comments. It is a part of Junit/Java doc changes which @idreestahir @Juzerhk  are working on. 

@RicardoUsbeck Is it OK if we take these comments as a part of future release activity(along with Junit)? Can you approve this request?

**Comment by RicardoUsbeck** [comment]
"@RicardoUsbeck Is it OK if we take these comments as a part of future release activity(along with Junit)? Can you approve this request?" Sure @prasanthhs , please open tickets for the issues. Thanks to @sepidetari @Suganya31 for the comments. Quite detailed. Please open tickets next time directly without approving, this will help the work flow. @prasanthhs if the changes are approved, feel free to merge yourself.

**Pull request merged**